src/grid/__tests__/grid.test.tsx

```diff
@@ -50,4 +50,61 @@ describe('Grid component', () => {
     });
     expect(wrapper.getElement().childElementCount).toBe(2);
   });
+
+  test('assigns data-testid to the columns', () => {
```

Comment:
Why do we need this test?

Reply:
This is also related to your second comment.
`data-testid` is not  an internal implementation details and we should test that the attribute is assigned correctly and is accessible to the customers.